exporter/azureblobexporter/README.md

```diff
@@ -0,0 +1,24 @@
+# Azure Blob Exporter
```

Comment:
We should [include status header](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10116) in each component README

Reply:
Done